plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,22 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Subsystem: schedulerSubsystem,
+			Name:      "scheduling_algorithm_predicate_evaluation",
+			Help:      "Scheduling algorithm predicate evaluation duration",
+			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
```

Comment:
So, I was trying to follow the existing conventions, the options that we used till now include histograms(with exponential buckets) and gauge. 

> but generally you need to run few experiments and understand current behavior (what is fast? what is OK? what is too slow?) to create appropriate metric.

Can you please elaborate on the part related to creating appropriate metric. I thought, we wanted to have a metric for predicate and priority evaluation time. Are you suggesting that we need to have something like:

"The predicate evaluation time should be less than < 1ms if X, Y, Z predicates & priorities are choosen?"

> It strongly depends on the thing that you want to measure

And the thing that we want to measure is predicate & priority evaluation times.